src/dailies.ts

```diff
@@ -591,3 +597,11 @@ function pantogram(): void {
   visitUrl("inv_use.php?pwd&whichitem=9573");
   visitUrl(`choice.php?whichchoice=1270&pwd&option=1&m=${m}&e=5&s1=5789,1&s2=706,1&s3=24,1`);
 }
+
+function combBeach(): void {
```

Comment:
Do we want to do combing as overdrunk turns if the ascend flag is turned on? Barf mpa is probably still higher right?

Reply:
I think for now we should only use people's free combs; people run garbo to run garbo, not to comb the beach.